include/irr/builtin/glsl/bxdf/common.glsl

```diff
@@ -108,6 +112,11 @@ irr_glsl_LightSample irr_glsl_createLightSample(in vec3 L, in irr_glsl_Anisotrop
     return irr_glsl_createLightSample(L,dot(interaction.isotropic.V.dir,L),interaction.T,interaction.B,interaction.isotropic.N);
 }
 
+vec3 irr_glsl_getTangentSpaceL(in irr_glsl_LightSample s)
+{
+    return vec3(s.TdotL, s.BdotL, s.NdotL);
+}
```

Comment:
where is this used?

Reply:
nowhere, just an utility

Reply:
delete it them

Reply:
ok.... why? XD

Reply:
if we dont use something no point keeping it around (we have git history after all)

Reply:
?

Reply:
why do we still have it?